plugins/modules/zos_job_query.py

```diff
@@ -72,12 +72,13 @@
 RETURN = r"""
 changed:
   description:
-     True if the state was changed, otherwise False.
+    True if the state was changed, otherwise False.
```

Comment:
I wonder if this doc is actually just copy/paste because in a query module (I need to check the code) do we ever change anything to cause the state to change?